### DISCUSSION TITLE
Any idea if it possible to recreate the health and stamina bars in the top left?

### DISCUSSION THREAD

**Discussion by kartikdc** [HunterPie/HunterPie#157] [Ideas]
Rise's look kinda ugly lmao

**Comment by haato3o** [comment]
That's actually planned already, in fact, I already have the UI for it finished, just missing the in-game data now

**Reply by kartikdc** [reply]
Thanks. Good luck.

**Reply by kartikdc** [reply]
Another suggestion: Post quest completion "returning to village" countdown timer that displays the actual number of seconds rather than an ambiguous hourglass
